### PR TITLE
[stable/drone] Add services permission to pipeline role

### PR DESCRIPTION
**Pull request by pedrovazsa**
#### What this PR does / why we need it:

- Bump drone version to RC7 release
- Add `services` to the pipeline service account role in [role-pipeline.yaml](stable/drone/templates/role-pipeline.yaml)

If the pipeline Service Account does not have `services` permission, Drone pipelines that require services will fail with an error similar to:
```
default: services is forbidden: User "system:serviceaccount:default:[release-name]-drone-pipeline" cannot create services in the namespace “ixjox3za1ed8vi8wspzgj"
```

This can be reproduced by running the current chart with any pipeline that uses services (e.g. [drone-demo-elasticsearch](https://github.com/drone-demos/drone-demo-elasticsearch)).

Adding `services` to the pipeline Service Account role fixes the issue.

#### Checklist

- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

**Comment by christian-roggia** [comment]
Once #11054 is merged you can bump the version and I will merge this PR. Thank you!

**Review by pedrovazsa — COMMENTED** [review]
Pleasure! :relaxed:  

Bumped chart version and updated app version.

File: stable/drone/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name: drone
 home: https://drone.io/
 icon: https://drone.io/apple-touch-icon.png
-version: 2.0.0-rc.6
-appVersion: 1.0.0-rc.4
+version: 2.0.0-rc.8
+appVersion: 1.0.0-rc.5
```
Comment (christian-roggia) [inline stable/drone/Chart.yaml:5]: The version of the app is still `1.0.0-rc.4`. Only the chart version should be bumped, you should also merge with the upstream repository to include the changes from the previously merged PR.

Comment (pedrovazsa) [reply stable/drone/Chart.yaml:5]: Merged with the upstream repository, should be good now. The app version seems to be `1.0.0-rc.5` on [Chart.yaml](https://github.com/helm/charts/blob/master/stable/drone/Chart.yamll) currently.

Cheers!

**Pull request merged**